Dockerfile.prod

```diff
@@ -16,6 +16,6 @@ FROM node:14.17-alpine
 USER node
 COPY --chown=node:node --from=build /app /app
 WORKDIR /app
-ENV NODE_ENV development
+ENV NODE_ENV production
 
-CMD ["npm", "run", "start:production"]
+CMD ["sh", "-c", "npm run start:${MICROS_GROUP}:production"]
```

Comment:
How many worker and web threads are we starting on each node? I guess the default is 1? Are we using the resources effectively with only 1 worker on each node?

Reply:
Good question. We might optimise it in the future by starting more threads. For now we can control the scale with the amount of nodes.

@mboudreau How many threads are used in Heroku?

Reply:
I think github said 8 for the API and 3-4 for the worker

Reply:
It seems our current docker images for heroku doesn't use threads (we use the same command). I think we need to make an issue for the further threads/nodes optimisation and move on

Reply:
Sorry, I misread that, thought you said nodes.  Threads depended on the size of their instance.  I just pushed a change to micros to use throng's default (it's a node clustering app) which will use the maximum number of threads - 1 as one needs the be the 'coordinator'.